docs/learning/SchemaUpgrade.md

```diff
@@ -0,0 +1,56 @@
+| Property   | Add     | :heavy_check_mark:| |
+| Property   | Delete     | :x:| |
+| Property   | Update     | :heavy_check_mark::grey_exclamation:|<table><tbody><tr><td>name</td><td>:x:</td></tr><tr><td>displayLabel</td><td>:heavy_check_mark:</td></tr><tr><td>description</td><td>:heavy_check_mark:</td></tr><tr><td>[custom attributes](#update_ca)</td><td>:heavy_check_mark::grey_exclamation:</td></tr><tr><td>type</td><td>:x:</td></tr><tr><td>kind of quantity</td><td>:heavy_check_mark:</td></tr><tr><td>category</td><td>:heavy_check_mark:</td></tr><tr></tbody></table>
+| StructClass   | Add     | :heavy_check_mark:| |
+| StructClass   | Delete     | :x:| |
+| CustomAttributeClass   | Add     | :heavy_check_mark:| |
+| CustomAttributeClass   | Delete     | :x:| |
+
+### Updating BaseClasses for EntityClass Rules<a id='update_mixins'></a>
+
+A EntityClass can have only one *BaseClass* of type EntityClass. It can also have zero or more Mixins. A Mixin is define similar to entity class but have addition custom attribute called <code>IsMixIn</code>. Mixin are not directly mapped. Its properties if any is merge into EntityClass that implement (inherits) from it. In some cases Mixins can be added or delete from base class list of a EntityClass but in other it cannot.
+
+| Type of BaseClass | Operation | Supported| Description|
+|---------------|----------|------------------|------
+|EntityClass | Update, Delete | :x: | Base class cannot be changed|
+|Mixin without properties | Add, Delete | :heavy_check_mark: | |
+|Mixin with properties | Add. Delete | :warning: | *Mix with properties is add only*<table><tbody><tr><td>add new mixin </td><td>:heavy_check_mark:</td><tr><td>delete existing mixin </td><td>:x:</td></tbody></table>|
+
+### Updating CustomAttributes Rules<a id='update_ca'></a>
+
+Following mapping custom attributes are not allowed to update with exception to <code>DbIndexList</code>.
+| Schema        | CustomAttribute Class | Operation  | Supported  | Description|
+|---------------|----------|------------------|--------------|--------------|
+| ECDbMap | SchemaMap| Update, Delete|:x:| No change allowed.
+| ECDbMap | ClassMap| Update, Delete|:x:| No change allowed.
+| ECDbMap | JoinedTablePerDirectSubclass| Update, Delete|:x:| No change allowed.
+| ECDbMap | ShareColumns| Update, Delete|:x:| No change allowed.
+| ECDbMap | PropertyMap| Update, Delete|:x:| No change allowed.
+| ECDbMap | ForeignKeyConstraint| Update, Delete|:x:| No change allowed.
+| ECDbMap | LinkTableRelationshipMap| Update, Delete|:x:| No change allowed.
+| ECDbMap | DbIndexList| Update, Delete|:heavy_check_mark::grey_exclamation:| Only non-unique indexes are allowed to be added or deleted.
```

Comment:
Why not use Unicode characters instead of emoji shortcodes? The latter may not display as expected everywhere (such as on our docsite).